### PR TITLE
LLM: Add peak diff

### PR DESCRIPTION
**Pull request by WeiguangHan**
@Oscilloscope98 Add the difference between current data and best history one.

### CODE REVIEW

**Comment by Oscilloscope98** [comment]
Please resolve the conflicts :)

File: python/llm/test/benchmark/csv_to_html.py [hunk 1]
```diff
@@ -96,9 +136,15 @@ def main():
...
+        latest_csv.insert(loc=7,column='best last1',value=best_last1)
+        latest_csv.insert(loc=8,column='best diff1(%)',value=best_diff1)
+        latest_csv.insert(loc=9,column='best last2',value=best_last2)
+        latest_csv.insert(loc=10,column='best diff2(%)',value=best_diff2)
+
+        subset=['diff1(%)','diff2(%)','best diff1(%)','best diff2(%)']
         columns={'1st token avg latency (ms)': '{:.2f}', '2+ avg latency (ms/token)': '{:.2f}', 'last1': '{:.2f}', 'diff1(%)': '{:.2f}',
-                'last2': '{:.2f}', 'diff2(%)': '{:.2f}', 'encoder time (ms)': '{:.2f}', 'peak mem (GB)': '{:.2f}'}
+                'last2': '{:.2f}', 'diff2(%)': '{:.2f}', 'encoder time (ms)': '{:.2f}', 'peak mem (GB)': '{:.2f}',
+                'best last1': '{:.2f}', 'best diff1(%)': '{:.2f}', 'best last2': '{:.2f}', 'best diff2(%)': '{:.2f}'}
```
Comment (Oscilloscope98) [inline python/llm/test/benchmark/csv_to_html.py:147]: Let's change the name to "best 1" and "best 2"; best last 1 and best last 2 is strange :)

Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py:147]: sure

**Comment by Oscilloscope98** [comment]
Closed as duplicated: https://github.com/intel-analytics/BigDL/pull/9789

**Comment by Oscilloscope98** [comment]
Closed as duplicated: https://github.com/intel-analytics/BigDL/pull/9789

**Pull request closed without merging**